src/components/CodeManagement/tests/ManageCodesTab.test.jsx

```diff
@@ -40,26 +48,36 @@ const initialState = {
 const ManageCodesTabWrapper = props => (
```

Comment:
nit: should we use object destructuring on these props so that `store` and `subsidyRequestConfiguration` don't get passed to `ManageCodesTab` below?